File: Clases_Csharp/GestorReportes.cs
using System;
using System.Collections.Generic;

namespace SistemaVeterinaria
{
    public class GestorReportes
    {
        public void GenerarReporteCitas(List<Cita> citas)
        {
            Console.WriteLine("--- REPORTE DE CITAS REGISTRADAS ---");
            foreach (var c in citas)
            {
                Console.WriteLine($"Fecha: {c.Fecha} | Hora: {c.Hora} | Mascota: {c.Mascota.Nombre} | Veterinario: {c.Veterinario.Nombre}");
            }
        }

        public void GenerarReporteInventarioBajo(List<Producto> inventario, int limiteMinimo)
        {
            Console.WriteLine("--- REPORTE DE PRODUCTOS CON STOCK BAJO ---");
            foreach (var p in inventario)
            {
                if (p.CantidadStock <= limiteMinimo)
                {
                    Console.WriteLine($"⚠️ Alerta: {p.Nombre} | Stock actual: {p.CantidadStock}");
                }
            }
        }
    }
}

File: Clases-Bluej/GestorReportes.java
import java.util.ArrayList;

public class GestorReportes {

    public void generarReporteCitas(ArrayList<Cita> citas) {
        System.out.println("--- REPORTE DE CITAS REGISTRADAS ---");
        for (Cita c : citas) {
            System.out.println("Fecha: " + c.getFecha() + " | Hora: " + c.getHora() + 
                               " | Mascota: " + c.getMascota().getNombre() + 
                               " | Veterinario: " + c.getVeterinario().getNombre());
        }
    }

    public void generarReporteInventarioBajo(ArrayList<Producto> inventario, int limiteMinimo) {
        System.out.println("--- REPORTE DE PRODUCTOS CON STOCK BAJO ---");
        for (Producto p : inventario) {
            if (p.getCantidadStock() <= limiteMinimo) {
                System.out.println("⚠️ Alerta: " + p.getNombre() + " | Stock actual: " + p.getCantidadStock());
            }
        }
    }
}